config/locales/models/report.en.yml

```diff
@@ -31,11 +31,16 @@ en:
   tabs:
     report:
       activities: Activities
+      forecasts: Forecasts
       variance: Variance
       budgets: Budgets
+
   page_content:
     reports:
       title: Reports
+    tab_content:
+      forecasts:
+        heading: Forecast in this report
```

Comment:
```suggestion
        heading: Forecasts in this report
```

Reply:
thanks, I think this makes sense, even though it deviates from Meyric's mockup.

We now show:

- tab name: "Forecasts", consistent with "Activities" and "Budgets" also plural
- page title: "Forecasts in this report" i.e. here is a list of the forecasts associated with this report
- heading of table: "Forecast spend per activity" - forecast is an adjective here, qualifying the noun "spend"

Reply:
Cool - that sounds fine 👍